connexion/apps/flask_app.py

```diff
@@ -100,6 +108,29 @@ def add_error_handler(self, error_code, function):
         # type: (int, FunctionType) -> None
         self.app.register_error_handler(error_code, function)
 
+    def route(self, rule, **options):
```

Comment:
Decorator to allow for registration of additional routes on the Connexion App instead of having to use the underlying Flask app. The same behavior is offered by the async app.

TODO: ccheck if this can be lifted to the AbstractApp, or copy documentation to Async App as well.